modules/jar-handling-utils/pom.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<modelVersion>4.0.0</modelVersion>
+<parent>
+    <groupId>org.mule.runtime</groupId>
+    <artifactId>mule-modules</artifactId>
+    <version>4.5.0-SNAPSHOT</version>
+</parent>
+
+<artifactId>mule-jar-handling-utils</artifactId>
+<packaging>jar</packaging>
+<name>Mule jar handling utils</name>
+<description>
+    This is a multi release module that allows handling of JDK internal jar classes needed for Java 8 runtime and a cleaner way in 11+.
+</description>
+
+<properties>
+    <formatterConfigPath>../../formatter.xml</formatterConfigPath>
+</properties>
+    <build>
+        <plugins>
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <version>${mavenCompilerVersion}</version>
+            <executions>
+                <execution>
+                    <id>default-compile</id>
+                    <goals>
+                        <goal>compile</goal>
+                    </goals>
+                    <configuration>
+                        <release combine.self="override"></release>
+                        <source>${javaVersion}</source>
+                        <target>${javaVersion}</target>
+                        <!-- recompile everything for target VM except the module-info.java -->
+                        <excludes>
+                            <exclude>module-info.java</exclude>
+                        </excludes>
+                    </configuration>
+                </execution>
+                <execution>
+                    <id>base-compile</id>
+                    <configuration>
+                        <release>11</release>
+                        <compileSourceRoots>
+                            <compileSourceRoot>${project.basedir}/src/main/java11</compileSourceRoot>
+                        </compileSourceRoots>
+                        <multiReleaseOutput>true</multiReleaseOutput>
+                    </configuration>
+                </execution>
+                <execution>
+                    <id>module-info-compile</id>
+                    <goals>
+                        <goal>compile</goal>
+                    </goals>
+                    <configuration>
+                        <release>11</release>
+                        <!-- recompile the module-info.java -->
+                        <includes>
+                            <include>module-info.java</include>
+                        </includes>
+                    </configuration>
+                </execution>
```

Comment:
there is no module-info in this module, why is ths needed?

Reply:
I was thinking we would add I can remove for now